DEPS

```diff
@@ -965,6 +1010,36 @@ hooks = [
       'src/flutter/tools/activate_emsdk.py',
     ]
   },
+  {
```

Comment:
Will this work for external users with no access to goma?

Reply:
No, but this runhook will only run if the `use_cipd_goma` is explicitly set to true.